src/tests/activity.test.js

```diff
@@ -0,0 +1,5 @@
+// Test basico para evitar conflictos.
```

Comment:
english please

Reply:
for ex:
```
// Basic test to avoid conflicts
// REMOVE THIS TEST AND MAKE YOURS!
test('should pass a basic test', () => {
  expect(1 + 1).toBe(2);
});
```